src/robot/libdocpkg/model.py

```diff
@@ -113,7 +113,7 @@ def to_dictionary(self):
             'name': self.name,
             'doc': self.doc,
             'version': self.version,
-            'generated': get_timestamp(daysep='-', millissep=None),
+            'generated': get_timestamp_for_doc(),
```

Comment:
This changes the current timestamp from `2022-05-27 22:07:15` to `2022-05-27T19:07:15+00:00`. The old timestamp was naive and I guess it makes sense to have the timezone information there. The change is only in HTML output so it shouldn't cause real backwards compatibility problems. I don't particularly like the `T` separator in outputs targeted for humans, but don't care strongly about it.

Reply:
Ooops, my comment was partly invalid. This changes also JSON spec files, not only HTML output. Thus this change is backwards incompatible the same way as the change to XML spec files. Also makes the `T` separator pretty much mandatory.